juniper_official/Solutions/EVPN-VXLAN/leaf-evpn-vxlan-kpis.playbook

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Same comments as "juniper_official/Solutions/EVPN-VXLAN/spine-evpn-vxlan-kpis.playbook"

Reply:
Added this in all playbooks